src/assemble_workflow/README.md

```diff
@@ -28,6 +28,53 @@ The following options are available in `assemble.sh`.
 | --keep             | Do not delete the temporary working directory on both success or error. |
 | -v, --verbose      | Show more verbose output.                                               |
 
+
+### run_assemble.py Options
+
+You can also run the assembling logic directly using Python:
+
+```bash
+python src/run_assemble.py <manifest-file> [options]
+```
```

Comment:
I do not think specifying this as we use the shell script as entrypoint with everything linked to pipenv.
This would add up confusions and remove the abstraction layer.
Users may not installed all the libs beforehand.